Integrations/integration-Moloch.yml

```diff
@@ -58,29 +59,94 @@ script:
         'moloch_unique_json': 'unique.txt',
     }
 
+    var prToName = {
+        1: 'icmp',
+        6: 'tcp',
+        17: 'udp',
+        47: 'gre',
+        58: 'icmpv6'
+    }
+
     var jsonResponses = ['moloch_connections_json', 'moloch_files_json', 'moloch_sessions_json', 'moloch_spigraph_json', 'moloch_spiview_json', 'moloch_unique_json'];
     var csvResponses = ['moloch_connections_csv', 'moloch_sessions_csv'];
     var pcapResponses = ['moloch_sessions_pcap'];
 
+    var parseSessions = function(response) {
+        var outputs = [];
+        var contexts = [];
+        for (var i = 0, objects = response.data; i < objects.length; i++) {
+            var output = {};
+
+            output['ID'] = objects[i].id;
+            output['Index'] = objects[i].index;
+
+            if (objects[i].pr in prToName) {
+                output['Protocol'] = prToName[objects[i].pr];
+            } else {
+                output['Protocol'] = objects[i].pr;
+            }
+
+            output['Start Time'] = convertTimestampToString(objects[i].fp);
+            output['End Time'] = convertTimestampToString(objects[i].lp);
+
+            output['Source IP'] = numToIp(objects[i].a1);
+            output['Source Port'] = objects[i].p1;
+            output['Source Packets'] = objects[i].pa1;
+            output['Source Bytes'] = objects[i].by1;
+            output['Source Databytes'] = objects[i].db1;
+            output['Destination IP'] = numToIp(objects[i].a2);
+            output['Destination Port'] = objects[i].p2;
+            output['Destination Packets'] = objects[i].pa2;
+            output['Destination Bytes'] = objects[i].by2;
+            output['Destination Databytes'] = objects[i].db2;
+
+            outputs.push(output);
+            contexts.push(convertKeysToPascalCase(output));
```

Comment:
Yep. Used it other integrations as well. The code is in CommonServer, you can take a look.